src/components/Network/NetworkGraphSelectFilters/NetworkGraphSelectFilters.tsx

```diff
@@ -0,0 +1,140 @@
+import { faChevronDown, faChevronUp } from '@fortawesome/pro-regular-svg-icons'
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import { useRef, useState } from 'react'
+import { useClickAway, useKey } from 'react-use'
+
+import { LoaderCard } from '@/components/Loader'
+import networkGraphConnectionProperties from '@/constants/networkGraphConnectionProperties'
+import networkGraphFilterMenu from '@/constants/networkGraphFilterMenu'
+
+interface NetworkGraphSidebarProps {
+    filters?: any
```

Comment:
Do we not know the filters beforehand? If we do it would be cleaner to add the correct definition here